icu4c/source/test/cintltst/cloctst.c

```diff
@@ -6665,6 +6667,145 @@ static void test_unicode_define(const char *namech, char ch, const char *nameu,
   }
 }
 
+static void checkTerminating(
+    const char* locale, const char* inLocale) {
```

Comment:
nit: FWIW, it looks like the *majority* of this file uses 4 spaces for indentation. For some reason the functions above seem to use only 2 spaces. 
The [ICU User Guide](https://unicode-org.github.io/icu/userguide/dev/codingguidelines.html#tabs-and-indentation) seems to says 4 spaces -- but I don't think that all the code actually does that though...

Reply:
Good catch!
Yes, Frank, please fix the indent to 4 spaces.
Looks like you just need to increase the indentation by another 2 on all new lines since the further relative indents appear to be fine already.

For this line itself, I would just move the arguments to the end of the previous line.